Disaster-Response-Platform/backend/Controllers/form_fields.json

```diff
@@ -1,74 +1,80 @@
 {
     "resource": {
         "fields": [
-            {"name": "_id", "label": "ID", "type": "text"},
-            {"name": "created_by", "label": "Created By", "type": "text"},
+            {"name": "description", "label": "Description", "type": "text"},
             {"name": "condition", "label": "Condition", "type": "select", "options": ["new", "used"]},
             {"name": "initialQuantity", "label": "Initial Quantity", "type": "number"},
-            {"name": "currentQuantity", "label": "Current Quantity", "type": "number"},
-            {"name": "type", "label": "Type", "type": "text"},
+            {"name": "type", "label": "Type", "type": "select", "options": ["cloth", "food", "shelter", "medication", "transportation", "tool", "human", "other"]},
             {"name": "x", "label": "X Coordinate", "type": "number"},
-            {"name": "y", "label": "Y Coordinate", "type": "number"}
+            {"name": "y", "label": "Y Coordinate", "type": "number"},
+            {"name": "occur_at", "label": "Occur At", "type": "date"},
+            {"name": "recurrence_rate", "label": "Recurrence Rate", "type": "select", "options": ["none", "daily", "every 2 days", "every 3 days", "weekly"]},
+            {"name": "recurrence_deadline", "label": "Recurrence Deadline", "type": "date"}
         ]
     },
     "need": {
         "fields": [
-            {"name": "_id", "label": "ID", "type": "text"},
-            {"name": "created_by", "label": "Created By", "type": "text"},
+            {"name": "description", "label": "Description", "type": "text"},
             {"name": "initialQuantity", "label": "Initial Quantity", "type": "number"},
-            {"name": "urgency", "label": "Urgency", "type": "number"},
-            {"name": "unsuppliedQuantity", "label": "Unsupplied Quantity", "type": "number"},
-            {"name": "type", "label": "Type", "type": "text"},
+            {"name": "urgency", "label": "Urgency", "type": "select", "options": [1, 2, 3, 4, 5]},
+            {"name": "type", "label": "Type", "type": "select", "options": ["cloth", "food", "shelter", "medication", "transportation", "tool", "human", "other"]},
             {"name": "x", "label": "X Coordinate", "type": "number"},
-            {"name": "y", "label": "Y Coordinate", "type": "number"}
+            {"name": "y", "label": "Y Coordinate", "type": "number"},
+            {"name": "occur_at", "label": "Occur At", "type": "date"},
+            {"name": "recurrence_rate", "label": "Recurrence Rate", "type": "select", "options": ["none", "daily", "every 2 days", "every 3 days", "weekly"]},
```

Comment:
maybe we dont need every 2 days and every3 days, can you provide a scenerio for it

Reply:
> maybe we dont need every 2 days and every3 days, can you provide a scenerio for it

I might want baby diapers for every 3 days, no?

Reply:
I think it can be provided accumulative, so no